easybuild/easyconfigs/s/SPH-EXA/SPH-EXA-develop-gompi-2021a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "CMakeMake"
+
+name = "SPH-EXA"
+version = "develop"
+homepage = "https://github.com/unibas-dmi-hpc/SPH-EXA_mini-app"
+description = """
+The SPH-EXA mini-app is a C++ scientific research code; its goal is to scale
+the Smoothed Particle Hydrodynamics (SPH) method to enable Tier-0 and Exascale
+simulations. More infos here: https://hpc.dmi.unibas.ch/en/research/sph-exa/
+"""
+toolchain = {"name": "gompi", "version": "2021a"}
+sources = [
+    {
+        "filename": "%(name)s.tar.gz",
+        "git_config": {
+            "url": "https://github.com/unibas-dmi-hpc",
+            "repo_name": "SPH-EXA_mini-app",
+            "tag": "develop",
+        },
+    }
+]
+checksums = [None]
+builddependencies = [
+    ("CMake", "3.20.1", "", ("GCCcore", "10.3.0")),
+]
```

Comment:
Could you add blank lines in between the different sections please? It's a bit hard to read as-is.